### PR TITLE
Conv1D Example

### PR DESCRIPTION
**Pull request by JesperDramsch**
Add Conv1D example with simple Sequential Model and input_shape

### CODE REVIEW

**Review by Kab1r — COMMENTED** [review]
Add line to import Sequential and Conv1D Layer

**Comment by JesperDramsch** [comment]
> Add line to import Sequential and Conv1D Layer

I compared this to other examples, where it wasn't used. Is using imports canonical then?

File: tensorflow/python/keras/layers/convolutional.py [hunk 1]
```diff
@@ -323,6 +323,13 @@ class Conv1D(Conv):
     kernel_constraint: Constraint function applied to the kernel matrix.
     bias_constraint: Constraint function applied to the bias vector.
 
+  Examples:
+    ```python
+    # Small convolutional model for 128-length vectors
+    model = Sequential()
+    model.add(Conv1D(32, 3, activation='relu', input_shape(, 128))
```
Comment (tanzhenyu) [inline tensorflow/python/keras/layers/convolutional.py]: typo.

File: tensorflow/python/keras/layers/convolutional.py [hunk 2]
```diff
@@ -323,6 +323,13 @@ class Conv1D(Conv):
     kernel_constraint: Constraint function applied to the kernel matrix.
     bias_constraint: Constraint function applied to the bias vector.
 
+  Examples:
+    ```python
+    # Small convolutional model for 128-length vectors
+    model = Sequential()
```
Comment (tanzhenyu) [inline tensorflow/python/keras/layers/convolutional.py]: I would wish to be consistent with the example like this:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/core.py#L553-L563
specifically, let user know what input shape and output shape is expected from the layer.

Comment (JesperDramsch) [reply tensorflow/python/keras/layers/convolutional.py]: Thank you. Adjusted the code accordingly.

Should there be an example for "variable timesteps", where input is `(None, 128)`?
Would be easy to add and probably help many users.

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**